raiden/network/transport/matrix.py

```diff
@@ -190,6 +190,8 @@ def start(
         self._raiden_service = raiden_service
 
         self._login_or_register()
+        if self._client._handle_thread:
```

Comment:
Can you please add a comment that explains why it's necessary to wait on the sync call so it doesn't get 'optimised' away at some point.